### ISSUE TITLE
Allow message scheduled enqueue time to be expressed with TimeSpan

### ISSUE THREAD

**Issue by SeanFeldman** [Azure/azure-service-bus-dotnet#160]
`Message.ScheduledEnqueueTimeUtc` accepts `DateTime`. There's a problem with this approach - calculated time is based of the local clock and not the broker lock. The client cannot know the server time either. Allow deferring message enqueue operation by providing delay as `TimeSpan` (similar to lock duration). With this approach the server will be using its own clock, w/o the need to worry for the client to be in sync.

**Comment by SeanFeldman** [comment]
@jtaubensee not sure if the broker support this or not, so could be more than just a client side.

**Comment by SeanFeldman** [comment]
Ping

**Comment by nemakam** [comment]
@SeanFeldman , do you expect the variance in machine clocks (for UTC) be considerably more than the variable in the latency of the send scheduled message request?

**Comment by nemakam** [comment]
@SeanFeldman , Is this an issue you actually encountered?

**Comment by nemakam** [comment]
Closing this issue as it is already opened the service side repo: Azure/azure-service-bus#40

**Comment by SeanFeldman** [comment]
@nemakam the server side is to allow this call. Should this stay in place to be added to the client?

**Comment by nemakam** [comment]
Makes sense. I'll reopen the case. Will close it if required after the initial triage on the other repo.
